src/ng/compile.js

```diff
@@ -255,12 +265,26 @@
  *    The `$transclude` function also has a method on it, `$transclude.isSlotFilled(slotName)`, which returns
  *    `true` if the specified slot contains content (i.e. one or more DOM nodes).
  *
+ * The controller can provide the following methods that act as life-cycle hooks:
```

Comment:
Should we mention that this is basically a new phase that happens before both pre and postLink?